### DISCUSSION TITLE
Mail options

### DISCUSSION THREAD

**Discussion by Ajayj231** [madrecha/jamku#593] [QA] [answered]
Hi,

2 requests regarding mail option.

1) Is it possible to have mailing option at service level, like got to compliance under particular service , select clients and send mail. Because , bulk information mails usually are sent as per service.

2) Under task list filter option can you add filter for profile, so that it is easier to get the mail or phone numbers or other profile details easily.

**Comment by adarshmadrecha** [comment]
> Is it possible to have mailing option at service level, like got to compliance under particular service , select clients and send mail. Because , bulk information mails usually are sent as per service.

We have this option already. Open Service and click on the Client Count.

![image](https://user-images.githubusercontent.com/11911938/122236375-18416d00-cedc-11eb-9b45-2a719746b928.png)

Then, in the Client List, Select all the Clients and send email.

![image](https://user-images.githubusercontent.com/11911938/122236662-52127380-cedc-11eb-9676-e934c4674a30.png)

**Comment by adarshmadrecha** [comment, marked as answer]
> 2.Under task list filter option can you add filter for profile, so that it is easier to get the mail or phone numbers or other profile details easily.

We have this feature already in Jamku.
To copy emails of the Client, select task and click on the Send Email Button. In the dialogue box go to "Copy Email" tab.

![image](https://user-images.githubusercontent.com/11911938/122237153-b9302800-cedc-11eb-8a60-7dc919dacc80.png)


If you wish to view any other profile details, copy the Client Numbers (using Table View)

![image](https://user-images.githubusercontent.com/11911938/122237609-162bde00-cedd-11eb-9796-746759490cdd.png)

and paste the Client Numbers in the Profile List 
![image](https://user-images.githubusercontent.com/11911938/122237843-44a9b900-cedd-11eb-9a19-13b2b5548a91.png)
